packages/react-app/src/components/ChallengeReviewList.jsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { List, Button, Input } from "antd";
+import { challengeInfo } from "../data/challenges";
+
+export default function ChallengeReviewList({ challengeSubmissions, isLoading, approveClick, rejectClick }) {
+  const [commentMap, setCommentMap] = React.useState({});
+  return (
+    <List
+      loading={isLoading}
+      itemLayout="vertical"
+      dataSource={challengeSubmissions}
+      renderItem={challenge => (
+        <List.Item
```

Comment:
![image](https://user-images.githubusercontent.com/2486142/133971891-22fad7b1-01cb-4fbf-97ef-843ece000b9c.png)

This looks a bit odd (too much space I think). Maybe we can try shrinking the container?